aws_doc_sdk_examples_tools/metadata_validator.py

```diff
@@ -185,12 +185,11 @@ def validate_files(
 
 
 def validate_metadata(doc_gen_root: Path, errors: MetadataErrors) -> MetadataErrors:
-    with open(Path(__file__).parent.parent / "config" / "sdks.yaml") as sdks_file:
+    config = Path(__file__).parent / "config"
```

Comment:
Is there a reason you're opening the files again instead of using the parsed DocGen versions?
Oh, is it because you're still using the yamale validation here and that needs plain dicts?

Reply:
Yamale. To remove later.